src/source/Signaling/LwsApiCalls.c

```diff
@@ -1493,6 +1494,9 @@ STATUS joinStorageSessionLws(PSignalingClient pSignalingClient, UINT64 time)
     PCHAR pResponseStr;
     UINT32 resultLen;
 
+    UNUSED_PARAM(pResponseStr);
+    UNUSED_PARAM(pLwsCallInfo);
+    UNUSED_PARAM(resultLen);
```

Comment:
These seem like something we should log. Why else are we having these variables be stored locally?